runtime/syscall-server/syscall_server_main.cpp

```diff
@@ -21,49 +21,83 @@ using namespace bpftime;
 // global context for bpf syscall server
 static syscall_context context;
 
+template <typename F, typename... Args>
+auto handle_exceptions(F &&f, Args &&...args) noexcept
+	-> decltype(f(std::forward<Args>(args)...))
+{
+	try {
```

Comment:
Will try-catch on such critical path make hooking more slower?

Reply:
Thr syscall server is not as performance critical as the runtime. So I think it's OK